thermal.snbt

```diff
@@ -0,0 +1,1201 @@
+	default_hide_dependency_lines: false
+	default_quest_shape: ""
+	filename: "thermal"
+	group: "3EBF424A99DA33B7"
+	icon: "thermal:lead_gear"
+	id: "21CD0B400036DF54"
+	order_index: 1
+	quest_links: [ ]
+	quests: [
+		{
+			description: ["The &eMachine Frame&r is the basis of all machines in the &5Thermal Series&r, so you may want to stock up!"]
+			id: "0AA8A23A683CFA57"
+			rewards: [{
+				amount: 250L
+				currency: "eternalcurrencies:coins"
+				id: "080D1B46E5F26528"
+				ignore_reward_blocking: true
+				type: "eternalcurrencies:currency"
+			}]
+			subtitle: "Craft a &eMachine Frame&r"
+			tasks: [{
+				id: "58E66C383739E43F"
+				item: "thermal:machine_frame"
+				type: "item"
+			}]
+			x: -3.0d
+			y: 2.0d
+		}
+		{
+			dependencies: ["0AA8A23A683CFA57"]
+			description: ["The Redstone Furnace works exactly like a normal furnace but is &nfaster and only requires power to smelt&r!"]
+			id: "5279965A3C00CD9F"
+			subtitle: "Craft a &eRedstone Furnace&r"
+			tasks: [{
+				id: "530D1ED26AE7835D"
+				item: "thermal:machine_furnace"
+				type: "item"
+			}]
+			x: 0.0d
+			y: 2.0d
+		}
+		{
+			dependencies: ["0AA8A23A683CFA57"]
+			description: ["The Pulverizer can be used to &nincrease the amount of resources&r obtained from ores and more!"]
+			id: "6C22DE8FD78A93AD"
+			subtitle: "Craft a &ePulverizer&r"
+			tasks: [{
+				id: "7AABE0A54DF46200"
+				item: "thermal:machine_pulverizer"
+				type: "item"
+			}]
+			x: -1.0d
+			y: 2.0d
+		}
+		{
+			dependencies: ["0AA8A23A683CFA57"]
+			description: [
+				"The Sawmill will give &n6 planks per log, instead of just 4&r! "
+				""
+				"It also generates sawdust as a byproduct, which can be useful for many things, including the <rainb>Legendary Cardboard Box&r!"
+			]
+			id: "0A13D9AFD4429192"
+			subtitle: "Craft a &eSawmill&r"
+			tasks: [{
+				id: "494F1563F829A97B"
+				item: "thermal:machine_sawmill"
+				type: "item"
+			}]
+			x: 3.0d
+			y: 2.0d
+		}
+		{
+			dependencies: ["0AA8A23A683CFA57"]
+			description: ["The Induction Smelter is a useful way to &nmake alloys and smelt resources&r!"]
+			id: "1D9E37BBD5F54394"
+			subtitle: "Craft an &eInduction Smelter&r"
+			tasks: [{
+				id: "3F32F0837C38F38C"
+				item: "thermal:machine_smelter"
+				type: "item"
+			}]
+			x: 1.0d
+			y: 2.0d
+		}
+		{
+			dependencies: ["0AA8A23A683CFA57"]
+			description: ["The Centrifugal Separator is most notably used for the production of bitumen from bituminous sand, but also has a few extra features!"]
+			id: "19D5DA2F39486B28"
+			subtitle: "Craft a &eCentrifugal Separator&r"
+			tasks: [{
+				id: "0D456B1EA11D7FB5"
+				item: "thermal:machine_centrifuge"
+				type: "item"
+			}]
+			x: 7.0d
+			y: 2.0d
+		}
+		{
+			dependencies: ["0AA8A23A683CFA57"]
+			description: [
+				"The Multiservo Press can be used to &ncraft gears, plates, coins, and to pack/unpack resources into their compressed/uncompressed forms&r."
+				""
+				"All you need are the &eDie's!&r"

```

Comment:
this is a weird grammatical thing but i think this is more correct? the vibes are anyway.
```suggestion
				"All you need are &eDies!&r"
```